packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -218,4 +218,31 @@ class ShorebirdFlutter {
     shorebirdEnv.flutterRevision = revision;
     useFlutterProgress.complete();
   }
+
+  /// Returns an human readable string from the received [flutterRevision] and
+  /// [flutterVersion].
+  ///
+  /// If [flutterRevision] is null, the current revision will be used.
+  /// If [flutterVersion] is null, it will be calculated from the current
+  /// revision using [getVersionString].
+  Future<String> humanReadableVersion({
```

Comment:
```suggestion
  /// If [flutterRevision] is null, the current revision will be used.
  /// If [flutterVersion] is null, it will be calculated from the current
  ///   revision using [getVersionString].
  Future<String> humanReadableVersion({
```